### PR TITLE
Adds Librarian Stuff

### PR DESCRIPTION
**Pull request by Geevies**
* Added two librarian alt-titles. Tech Support, which handles.. IT stuff, such as resetting devices, and Curator, which handles display of artifacts, as well as pricing things with their price scanner.
* Added the tech support card, which can reset devices to their original state, as well as vend devices from the laptop vendor for free.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by alsoandanswer** [comment]
shouldn't "custodian" be "curator"?

File: code/modules/modular_computers/hardware/hard_drive.dm [hunk 1]
```diff
@@ -174,3 +174,16 @@
 			var/datum/computer_file/data/F = SSdocs.create_file(file)
 			store_file(F)
 	. = ..()
+
+/obj/item/computer_hardware/hard_drive/proc/reset_drive()
+	for(var/datum/computer_file/F in stored_files)
+		remove_file(F)
+	install_default_programs()
+	store_file(new /datum/computer_file/program/ntnetdownload(src))
```
Comment (Arrow768) [inline code/modules/modular_computers/hardware/hard_drive.dm]: The client and hardware manager applications are usually part of the default programs.
This allows to determine if the device is a company (no software downloader) or a private device.
Remove the manual addition of the downloader here.

**Comment by alsoandanswer** [comment]
yeah, went and checked

"custodian" can mean 
chrome_Z9JcLMtyH2

"curator" seems much more classy, and actually means what you want
chrome_VTdrUnUqSc

**Pull request merged**